framework/src/node/bft/endpoint.ts

```diff
@@ -35,4 +37,16 @@ export class BFTEndpoint extends BaseEndpoint {
 		}
 		return areDistinctHeadersContradicting(bftHeader1, bftHeader2);
 	}
+
+	public async getGeneratorList(ctx: ModuleEndpointContext): Promise<{ list: string[] }> {
+		const votesStore = ctx.getStore(this.moduleID, STORE_PREFIX_BFT_VOTES);
+		const bftVotes = await votesStore.getWithSchema<BFTVotes>(EMPTY_KEY, bftVotesSchema);
+		const { height: currentHeight } =
+			bftVotes.blockBFTInfos.length > 0 ? bftVotes.blockBFTInfos[0] : { height: 1 };
```

Comment:
Should it maybe be `bftVotes.blockBFTInfos[0].height` instead of just `bftVotes.blockBFTInfos[0].height`?

Reply:
`bftVotes.blockBFTInfos[0]` will return
```
{
	height: number;
	generatorAddress: Buffer;
	maxHeightGenerated: number;
	maxHeightPrevoted: number;
	prevoteWeight: bigint;
	precommitWeight: bigint;
}
```
and it is destructured on the left hand, so this is correct

Reply:
Ah, I missed the destructing. Also not so familiar with that syntax, all good then!